content/applications/general/iot/config/troubleshooting.rst

```diff
@@ -343,6 +343,153 @@ Save`.
+
+.. image:: troubleshooting/modify-select-printer.png
+   :align: center
+   :alt: Printer selection screen with Continue highlighted.
+
+On the next page, click :guilabel:`Continue` to proceed to set the :guilabel:`Make` of the printer.
+
+.. image:: troubleshooting/modify-printer-dymo.png
+   :align: center
+   :alt: Printer modification screen with Continue highlighted.
+
+Under :guilabel:`Make` select :guilabel:`DYMO` from the menu. Click on :guilabel:`Continue` to set
+the :guilabel:`Model`.
+
+.. image:: troubleshooting/setting-make.png
+   :align: center
+   :alt: Setting the make page, with DYMO and continue highlighted.
+
+On the following page, set the :guilabel:`Model` to :guilabel:`DYMO LabelWriter 450 DUO Label (en)`
+(or whichever DYMO printer model is being used). Click on :guilabel:`Modify Printer` to complete
+setting the new driver, a confirmation page will appear.
+
+.. image:: troubleshooting/setting-model.png
+   :align: center
+   :alt: Setting the printer model page with DYMO LabelWriter 450 DUO Label (en) highlighted.
+
+After being redirected to a confirmation page, acknowledging a successful update, click on the
+:guilabel:`Printers` button in the top menu.
+
+All the printers installed on the OpenPrinting CUPS server appear, including the newly updated:
+:guilabel:`DYMO LabelWriter 450 DUO Label` (or whichever DYMO printer model is being used). Click
+into the printer that was just updated.
+
+To print a test label, click on the :guilabel:`Maintenance` drop-down menu to the left of the
+:guilabel:`Administration` drop-down menu, and select :guilabel:`Print Test Page`. The test label
+will print out with a ten-second delay if the driver update was successful.
+
+.. image:: troubleshooting/print-test.png
+   :align: center
+   :alt: Printing a test page from the administration drop-down menu in the OpenPrinting CUPs
+         server.
+
+To reduce this delay a new printer will need to be added, follow the process below.
+
+DYMO LabelWriter print delay
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To resolve the delay issue after modifying the driver, the printer **must** be reinstalled. To
+reinstall the printer, open the OpenPrinting CUPS administration page by clicking
+:guilabel:`Printers server`, at the bottom of the :abbr:`IoT (Internet of Things)` box homepage.
+Then, click on :guilabel:`Administration` in the top menu, then click :guilabel:`Add a Printer`.
+
+.. tip::
+   If the DYMO LabelWriter 450 DUO printer is not printing at all, or is not recognizable (has a RAW
+   driver type), then update the drivers on the device. See
+   :ref:`troubleshooting/dymo/update_drivers`.
+
+.. image:: troubleshooting/add-printer-dymo.png
+   :align: center
+   :alt: Add a printer button highlighted on the Printer CUPS management page.
+
+On the next screen, in the :guilabel:`Local Printers` section, select the :guilabel:`DYMO
+LabelWriter 450 DUO Label (DYMO LabelWriter 450 DUO Label)` (or whichever DYMO printer model is
```

Comment:
Just double checking that the same text outside the parenthesis are inside "(DYMO LabelWriter 450 DUO Label)", is this like that in the UI?

Reply:
Yes this is in the UI

![modify-select-printer](https://github.com/odoo/documentation/assets/91896312/f4b6b008-de6c-4f5e-aacf-8391f76efca6)